test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -325,3 +319,49 @@ UCS_TEST_P(test_uct_sockaddr, conn_to_non_exist_server)
 }
 
 UCT_INSTANTIATE_SOCKADDR_TEST_CASE(test_uct_sockaddr)
+
+class test_uct_cm_sockaddr : public uct_test {
+public:
+    void init() {
+        uct_test::init();
+
+        /* If we reached here, the interface is active, as it was tested at the
+         * resource creation */
+        if (!ucs::is_inet_addr((struct sockaddr *)&(GetParam()->connect_if_addr))) {
+            UCS_TEST_SKIP_R("There is no IP on the interface");
+        }
+
+        /* Get a usable port on the host */
+        uint16_t port = ucs::get_port();
+
+        /* This address is accessible, as it was tested at the resource creation */
+        listen_sock_addr.addr =
+            ucs::sockaddr_set_port(GetParam()->listen_if_addr, port);
+        connect_sock_addr.addr =
+            ucs::sockaddr_set_port(GetParam()->connect_if_addr, port);
+
+        server = uct_test::create_entity();
+        m_entities.push_back(server);
+        client = uct_test::create_entity();
+        m_entities.push_back(client);
+    }
+
+protected:
+    entity *server, *client;
+    ucs_sock_addr_t listen_sock_addr, connect_sock_addr;
+};
+
+UCS_TEST_P(test_uct_cm_sockaddr, cm_open_close)
+{
+    UCS_TEST_MESSAGE << "Testing " << ucs::sockaddr_to_str(listen_sock_addr.addr)
+                     << " Interface: " << GetParam()->dev_name;
+
+    for (size_t i = 0; i < m_entities.size(); ++i) {
+        uct_cm_attr_t attr;
+        ucs_status_t status = uct_cm_query(m_entities.at(i).cm(), &attr);
+        ASSERT_UCS_OK(status);
```

Comment:
question: will it `abort()` gtest and testing will be stopped?

Reply:
It will skip the test if `status == UCS_ERR_UNSUPPORTED`. do you think we need another macro?

Reply:
> do you think we need another macro?

I thought that this macro aborts testing - when `abort()` is called, the process is frozen and testing hangs.
If it just skip the test, `ASSERT_UCS_OK` is good here.